PickMeUp/PickMeUp/Caching/PMCachingFunctions.m

```diff
@@ -0,0 +1,73 @@
+//
+//  PMCachingFunctions.m
+//  PickMeUp
+//
+//  Created by Abigail Shilts on 7/26/22.
+//
+
+#import <Foundation/Foundation.h>
+#import "Parse/Parse.h"
+#import "PMCachingFunctions.h"
+#import "PMConversation.h"
+#import "StringsList.h"
+
+@interface PMCachingFunctions ()
+@end
+
+@implementation PMCachingFunctions
+
+static const NSString *const kConversationCacheFile = @"ConversationCache.plist";
```

Comment:
What's the reason to use a plist file instead of keychain or userdefaults?

Reply:
I think file storage (plist) is a reasonable tool for this purpose. Keychain and NSUserDefaults are normally used to store  a smaller piece of data and access it by key later. CoreData uses file store to store the data.